.github/workflows/uploadArchives.yml

```diff
@@ -0,0 +1,39 @@
+name: Upload Archives
+
+on:
+ push:
+   tags: '[0-9]+.[0-9]+.[0-9]+.*'
+
+jobs:
+ build:
+   env: 
+     ANDROID_API_LEVEL: 23
+     ANDROID_EMULATOR_LEVEL: 23
+     ANDROID_BUILD_TOOLS_VERSION: 28.0.3
+     ANDROID_ABI: armeabi-v7a
+     ANDROID_TAG: google_apis
+     ADB_INSTALL_TIMEOUT: 20
+
+   runs-on: linux-latest
+
+   steps:
+   - uses: actions/checkout@v2
+   - name: set up JDK 11
+     uses: actions/setup-java@v2
+     with:
+       java-version: 11.0.10
+       distribution: 'adopt'
+       cache: gradle
+
+   - name: Checkout submodules
+     uses: actions/checkout@v2
+     with:
+       fetch-depth: 2
+       submodules: recursive
+       token: ${{ secrets.ACCESS_TOKEN }}
+       persist-credentials: false
+
+   - name: Grant execute permission for gradlew
+     run: chmod +x gradlew
+   - name: Upload archives
+     run: ./gradlew uploadArchives
```

Comment:
not sure if I know where is the uploadArchives job code. Can you give me a reference to the code?